src/com/epam/javatrainings/annotation/Validator.java

```diff
@@ -0,0 +1,55 @@
+package com.epam.javatrainings.annotation;
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Validator {
+
+    void validate(CustomerDto instance) {
+        try {
+            Field f = instance.getClass().getDeclaredField("name");
+            Field email = instance.getClass().getDeclaredField("email");
+            Field birthDay = instance.getClass().getDeclaredField("birthDay");
+            Field discount = instance.getClass().getDeclaredField("discountRate");
+            if (f.isAnnotationPresent(Length.class)) {
+                f.setAccessible(true);
+                String name = (String) f.get(instance);
+                int nameFieldLen = name.length();
+                Length length = f.getAnnotation(Length.class);
+                if (nameFieldLen < length.min() || nameFieldLen > length.max()) {
+                    System.out.println("You entered wrong length please fix it");
```

Comment:
In requirements is written that we need to return a List of Errors, please create a dedicated class for it, and return from validate method List<Error>